cmd/argocd/commands/root.go

```diff
@@ -17,6 +17,7 @@ func init() {
 }
 
 func initConfig() {
+	cli.SetLogOutput()
```

Comment:
I think it would be better to keep sending logs to stderr for cli
Otherwise output of cli will be mixed with the logs